app/views/transactions/_form.html.slim

```diff
@@ -46,6 +53,13 @@
   - else
     = f.input :bank_account, disabled: true
 
+  - if params[:invoice_id] || f.object.invoice_id
+    = f.input :comission, input_html: { class: 'comission' }
```

Comment:
@yunixon please add `Amount` in separated line on this form (not like text input, just like a text, but in correct currency and format), so user will see Amount of new transaction, that he is going to create on this form.